content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -407,7 +407,7 @@ in the Pod Lifecycle documentation.
 To understand the context for why Kubernetes wraps a common Pod API in other resources (such as {{< glossary_tooltip text="StatefulSets" term_id="statefulset" >}} or {{< glossary_tooltip text="Deployments" term_id="deployment" >}}), you can read about the prior art, including:
 
 * [Aurora](https://aurora.apache.org/documentation/latest/reference/configuration/#job-schema)
-* [Borg](https://research.google.com/pubs/pub43438.html)
+* [Borg](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/43438.pdf)
```

Comment:
I'd link to https://research.google/pubs/large-scale-cluster-management-at-google-with-borg/

Reply:
I found that link by searching Google for the title of the PDF (Large-scale cluster management at Google with Borg)